.travis.yml

```diff
@@ -42,12 +42,12 @@ install:
   - pipenv install --skip-lock "six>=1.11.0"
   # TODO: temporary, remove when django-netjsonconfig 0.9.0 is released
   - if [[ $TRAVIS_PYTHON_VERSION == "2.7" ]]; then pipenv install git+git://github.com/tinio/pysqlite.git@extension-enabled#egg=pysqlite; fi
-  # TODO: temporary, remove when django-x509 is released with the passphrase add-on
-  - pipenv install --skip-lock https://github.com/openwisp/django-x509/tarball/master
+  - pipenv run pip install https://github.com/openwisp/django-x509/tarball/master
```

Comment:
I was having issues with the order of installation (when `django-netjsonconfig` is installed, it pulls the latest PyPi version of `django-x509`, and replaces the version installed from the tarball). After some experimentation with the order of lines in the `Pipfile`, it looks like this was the only solution that results in the latest versions of both `django-x509` and `django-netjsonconfig`.

You're right that I shouldn't really be installing it with pip, but I haven't found a way to prevent the issue without it.

Reply:
If you're interested, I added `pipenv run pip freeze` before and after some commands to see where the packages were getting replaced, but still couldn't figure out a solution that doesn't involve using `pipenv run pip`. See [my many failing build tests](https://travis-ci.org/openwisp/openwisp-controller/pull_requests) for reference.

Reply:
No problem, I just asked for the sake of completeness. I guess I can't come up with a solution for that quickly, either.

Reply:
It looks like specifying package installation order is a pending feature:
https://github.com/pypa/pipenv/issues/2274

Reply:
I don't mind that workaround for now, this is kind of similar to explicitely installing `django-netjsonconfig` from Github instead of the latest release from PyPi.